packages/account-v2/src/components/CountrySelector/CountrySelector.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useResidenceList } from '@deriv/api';
+import { LabelPairedChevronDownMdRegularIcon } from '@deriv/quill-icons';
+import { Dropdown } from '@deriv-com/ui';
+
+type TCountrySelector = {
+    errorMessage?: string;
+    label: string;
+};
+
+const CountrySelector = ({ errorMessage, label }: TCountrySelector) => {
+    const { data: residenceList } = useResidenceList();
+
+    return (
+        <Dropdown
+            dropdownIcon={<LabelPairedChevronDownMdRegularIcon />}
+            errorMessage={errorMessage}
+            label={label}
+            list={residenceList}
+            name='country'
+            /*eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
This will be removed once the functionality is implemented. Currently this is here because onSelect is required for the Dropdown component.

Reply:
Shouldn't this onSelect be coming from props ?

Reply:
Yes it should be a props. But for now we are not implementing any logics from anywhere we will leave it for now. Once we starting to import and use the component, then we should edit it in that PR